### PR TITLE
Add a cache-consumer chart

### PR DESCRIPTION
**Pull request by ericvaandering**
Fortunately this one will stay named "cache-consumer". 

@rcarpa how should we interact with #68? Should we just make this new one the new style of config only?

### CODE REVIEW

File: charts/rucio-daemons/templates/cache-consumer.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    {{- if .Values.serviceAccountName }}
+      serviceAccountName: {{ .Values.serviceAccountName }}
+    {{- end }}
+      volumes:
+      - name: config
+        secret:
+          secretName: {{ template "rucio.fullname" . }}.config.cache-consumer
+      - name: proxy-volume
+        secret:
+          secretName: {{ .Release.Name }}-rucio-x509up
```
Comment (rcarpa) [inline charts/rucio-daemons/templates/cache-consumer.yaml:69]: Does the cache consumer need the crtificate an ca bundle bellow ?

**Review by rcarpa — COMMENTED** [review]
I agree with you: I would directly remove the support for exporting the config to env from this daemon. Nobody should use that function anymore.

Other than that, looks good to me.

File: charts/rucio-daemons/templates/cache-consumer.yaml (hunk 1 above)
Comment (ericvaandering) [reply charts/rucio-daemons/templates/cache-consumer.yaml:69]: Oh, probably not. Let me see what makes sense here. I should have started with the Kronos setup as they are similar in function (I think)

**Comment by ericvaandering** [comment]
@rcarpa want to give it another look and then merge it?

**Review by rcarpa — APPROVED** [review]

**Pull request merged**